ports/d3dx12/vcpkg.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "d3dx12",
+  "version-string": "12aug2020",
```

```suggestion
  "version-string": "2020-09-05",
```
It seems that there is no version for this port. So we can use a date here. Please see https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/control-files.md#version.